<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelEXT">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Extérieur</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pression</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Température</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidité</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Humidité</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Temp Min / Max</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPression">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>210</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_8">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="lineEditIntTemp"/>
   </widget>
   <widget class="QLineEdit" name="lineEditExtHumidity">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIntHumidity">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="lineEditExtMin"/>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditExtMax"/>
   </widget>
   <widget class="QSplitter" name="splitter_9">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>159</width>
      <height>24</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelFormat">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>° C</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxF">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>° F</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelActuel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>              AUJOURD'HUI</string>
    </property>
   </widget>
   <widget class="QLabel" name="soleil">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>label soleil</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>540</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLineEdit" name="lineEditPrev1Temp"/>
    <widget class="QLineEdit" name="lineEditPrev2Temp"/>
    <widget class="QLineEdit" name="lineEditPrev3Temp"/>
    <widget class="QLineEdit" name="lineEditPrev4Temp"/>
    <widget class="QLineEdit" name="lineEditPrev5Temp"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="pict1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="pict2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="pict3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="pict4">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="pict5">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelPrevision">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>              PREVISIONS</string>
    </property>
   </widget>
   <widget class="QLabel" name="Date_Time">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="Digital_clock">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Heure</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelINT">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Intérieur</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="lineEditPrev1Date">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lineEditPrev2Date">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lineEditPrev3Date">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lineEditPrev4Date">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="lineEditPrev5Date">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>460</y>
      <width>881</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QLabel" name="fond">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFormat_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Température en :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>fond</zorder>
   <zorder>labelEXT</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>lineEditPression</zorder>
   <zorder>splitter_8</zorder>
   <zorder>lineEditExtHumidity</zorder>
   <zorder>lineEditIntHumidity</zorder>
   <zorder>splitter_7</zorder>
   <zorder>splitter_9</zorder>
   <zorder>labelFormat</zorder>
   <zorder>checkBoxF</zorder>
   <zorder>labelActuel</zorder>
   <zorder>soleil</zorder>
   <zorder>splitter_2</zorder>
   <zorder>splitter_3</zorder>
   <zorder>labelPrevision</zorder>
   <zorder>Date_Time</zorder>
   <zorder>Digital_clock</zorder>
   <zorder>labelINT</zorder>
   <zorder>splitter</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>labelFormat_2</zorder>
   <zorder>horizontalSpacer</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
